spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/ActuatorOperations.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.deployer.spi.app;
+
+/**
+ * Deployers may implement this extension to invoke actuator endpoints of deployed app instances.
+ *
+ * @author David Turanski
+ */
+public interface ActuatorOperations {
+
+	/**
+	 * Get a resource from an actuator path.
+	 * @param deploymentId the deployment ID of the deployed app.
+	 * @param guid unique id for the app instance.
+	 * @param endpoint the endpoint path relative to the base actuator URL for the instance.
```

Comment:
**[NIT]** I know that the impl handles the path w/ or w/o a slash but as a developer its always confusing for me to figure out to pass a path in w/ or w/o the slash unless the api tells me one way or the other. Wdyt about mentioning something about the expected format of the path/slash.

Reply:
I'm a big fan of Postel's law. I can update the comment.